dietpi/dietpi-software

```diff
@@ -12091,6 +12098,73 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 
 			unset -v choice_required domain port invalid_text server_ip
 		fi
+
+		if To_Install 213 # soju
+		then
+			local ver=0.8.0
+			aDEPS=('gcc' 'make' 'pkg-config' 'scdoc' 'libpam-dev' 'libsqlite3-dev')
+			Download_Install "https://git.sr.ht/~emersion/soju/archive/v${ver}.tar.gz"
+			G_EXEC cd "soju-v${ver}"
+
+			# Build
+			G_EXEC_OUTPUT=1 G_EXEC make all "-j$(nproc)" \
+				config_path=/mnt/dietpi_userdata/soju/config
+				GOFLAGS=-tags=pam,libsqlite3 \
```

Comment:
The reason for the shellcheck annotation is a syntax error, escaped line break at wrong line:
```suggestion
			# Build
			G_EXEC_OUTPUT=1 G_EXEC make all "-j$(nproc)" \
				config_path='/mnt/dietpi_userdata/soju/config' \
				GOFLAGS='-tags=pam,libsqlite3'
```